pyigm/guis/igmguesses.py

```diff
@@ -1236,22 +1249,29 @@ def on_draw(self, replot=True, in_wrest=None, rescale=True, fig_clear=False):
                     # Any lines inside?
                     mtw = np.where((line_wvobs > wvmnx[0]) & (line_wvobs < wvmnx[1]))[0]
                     for imt in mtw:
-                        v = c_kms * (line_wvobs[imt]/wvobs - 1)
+                        # v = c_kms * (line_wvobs[imt]/wvobs - 1)
+                        v = ltu.give_dv(line_wvobs[imt]/wrest - 1., self.z)
                         if self.flag_colorful:
                             color_label = line_color[imt]
                         else:
                             color_label = COLOR_MODEL
 
                         if self.flag_idlbl:
-                            self.ax.text(v, 0.5, line_lbl[imt], color=color_label, backgroundcolor='w',
+                            self.ax.text(v.value, 0.5, line_lbl[imt], color=color_label, backgroundcolor='w',
                                 bbox={'pad':0,'edgecolor':'none', 'facecolor':'w'}, size='xx-small',
                                     rotation=90.,ha='center',va='center')
 
                         if (self.flag_plotmodel) and (self.flag_colorful):
                             # plotting absline with color better done in wobs -> velo space
-                            dvmin = c_kms * ((line_wvobs_lims[imt][0] - line_wvobs[imt]) / wvobs)
-                            dvmax = c_kms * ((line_wvobs_lims[imt][1] - line_wvobs[imt]) / wvobs)
-                            cond = (velo.value >= v + dvmin) & (velo.value <= v + dvmax)
+                            # dvmin = c_kms * ((line_wvobs_lims[imt][0] - line_wvobs[imt]) / wvobs)
+                            # dvmax = c_kms * ((line_wvobs_lims[imt][1] - line_wvobs[imt]) / wvobs)
+                            vmin = ltu.give_dv(line_wvobs_lims[imt][0]/wrest - 1., self.z)
```

Comment:
Might be able to do with 1 call to give_dv(), i.e.

vmin, vmax = ltu.give_dv(....)